packages/devtools_app/lib/src/screens/app_size/app_size_controller.dart

```diff
@@ -246,7 +246,13 @@ class AppSizeController {
     // Build a tree with [TreemapNode] from [processedJsonMap].
     final newRoot = generateTree(processedJson)!;
 
-    changeAnalysisRoot(newRoot);
+    // Open Treemap to main app.
```

Comment:
For clarity: let's rename `newRoot` to `jsonRoot`.  That way we can do something like this:

```
    // Determine the correct root node.
    final newRoot = isDeferredApp
        ? jsonRoot.childrenMap.values.firstWhere((node) => node.name == 'Root')
        : jsonRoot;
    changeAnalysisRoot(newRoot);
```